src/style-spec/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 13.23.1-beta.1
+
+### 🐞 Bug fixes
```

Comment:
```suggestion
### ✨ Features and improvements
```
Nit: IMO this is a feature rather than a bug fix, but you could argue either way.

Reply:
Thanks for the input @SnailBones  - I'm OK with making it an improvement. Should the version instead be 13.24.0-beta.1?